VSTPlugin.cpp

```diff
@@ -85,6 +85,11 @@ void VSTPlugin::loadEffectFromPath(std::string path)
 			return;
 		}
 
+		if (!(effect->flags & effFlagsCanReplacing)) {
+			blog(LOG_WARNING, "VST Plug-in can't support replacing");
```

Comment:
Can you log the plugin path along with this warning?

Reply:
Yes, log content has been updated